gensim/__main__.py

```diff
@@ -0,0 +1,19 @@
+import platform
+import sys
+import numpy
+import scipy
+import gensim
+import argparse
+
```

Comment:
Missing docstring for the module, should looks like https://github.com/RaRe-Technologies/gensim/commit/117d447087b2e9a36fda1279d837e045fefd1558#diff-cbaf05cc913e951e3205d150e33edf2eR9

Reply:
In addition, don't forget about `.. program-output:: ` section

Reply:
Also, need to add `.rst` to `gensim/docs/src`

Reply:
On it

Reply:
> Also, need to add .rst to gensim/docs/src

How do I do this?

Reply:
Look into `gensim/docs/src/scripts` for example, you'll see several `*.rst` files, you need to add something similar to needed place (same as in package hierarchy).